### DISCUSSION TITLE
Highlight blacklisted customer/organization

### DISCUSSION THREAD

**Discussion by nick-traeger** [YetiForceCompany/YetiForceCRM#14314] [Feature improvement]
Hello,
we currently have a customer which we don't want to handle anymore. We can of course now set the status of the customer to Blacklisted, but it would still be nice to have some kind of warning about these customers. Maybe we could choose a color when editing the selection in the administration, which would colorize the row in the overview. The employee would then see at a glance that he has to be careful with this customer.

**Comment by alexgrndprx** [comment]
Not full row, but it's something, and, most importantly, its already implemented
![acc1](https://user-images.githubusercontent.com/79980057/109808065-59fd1f00-7c2f-11eb-9a38-90501f32f666.PNG)
![acc2](https://user-images.githubusercontent.com/79980057/109808063-58cbf200-7c2f-11eb-8137-6dbb7f6c779a.PNG)
